nixos/modules/hardware/openrazer.nix

```diff
@@ -49,7 +49,10 @@ in
 {
   options = {
     hardware.openrazer = {
-      enable = mkEnableOption "OpenRazer drivers and userspace daemon";
+      enable = mkEnableOption ''
+        OpenRazer drivers and userspace daemon.
+        Any users that wish to make use of the daemon need to be
+        members of the "plugdev" group.'';
```

Comment:
```suggestion
        members of the "plugdev" group.
        '';
```
otherwise LGTM

Reply:
one of these days I'll submit a PR without any formatting fixes required :sweat_smile: